### ISSUE TITLE
question about mask = tf.logical_and(mask, feat_scores > -0.5)

### ISSUE THREAD

**Issue by lihaixiang** [balancap/SSD-Tensorflow#149]
in ssd_common.py 
why "feat_scores > -0.5"， according to the paper,it should be "feat_scores > 0.5", i do not understand.

**Comment by HiKapok** [comment]
me too, I'm very confused about jaccard score > -0.5. have you figure out what does this mean? thank you.

**Comment by Mooonside** [comment]
the same issue appears in ssd_losses as well, but according to the definition of jaccard index, feat_scores is always larger than 0. So i guess this statement basically does nothing useful, maybe the author merely uses it to eliminate invalid ground truth bboxes?  P.s. @lihaixiang As for the  mentioned "feat_scores > 0.5" part, the positive labels is actually assigned in ssd_losses function.

**Comment by HiKapok** [comment]
I have modified this code a lot to train another similar detector, if you would like fix this code maybe you can refer to this [RON_Tensorflow](https://github.com/HiKapok/RON_Tensorflow/)
